server/views/components/button-link/index.njk

```diff
@@ -0,0 +1,7 @@
+{# button-link component data: modifierClasses, href, text, icon #}
+<a class="btn {{modifierClasses}}" href="{{href}}">
```

Comment:
I prefer `href` - but we should be consistent - not for now - but lets change it [here](https://github.com/wellcometrust/wellcomecollection.org/blob/master/server/views/components/header/index.njk#L25).